docs/community/how_to_release.md

```diff
@@ -0,0 +1,508 @@
+[RESULT][VOTE] Release Apache Fury(incubating) ${release_version}-${rc_version}
+```
+
+Content:
+
+```
+Hello, Apache Fury(incubating) Community,
+
+The vote to release Apache Fury(Incubating) ${release_version}-${rc_version} has passed.
+
+The vote PASSED with 3 binding +1 and 0 -1 vote:
+
+Binding votes:
+
+- xxx
+- yyy
+- zzz
+
+Vote thread: ${vote_thread_url}
+
+Thanks
+
+${name}
+```
+
+### Incubator Community Vote
+
+Send an email to: general@incubator.apache.org:
+
+Title:
+
+```
+[VOTE] Release Apache Fury(incubating) ${release_version}-${rc_version}
+```
+
+Content:
+
+```
+Hello Incubator PMC,
+
+The Apache Fury community has voted and approved the release of Apache
+Fury(incubating) ${release_candicate_version}. We now kindly request the IPMC members
+review and vote for this release.
+
+Apache Fury(incubating) - A blazingly fast multi-language serialization
+framework powered by JIT and zero-copy.
+
+Fury community vote thread:
+
+${community_vote_thread_url}
+
+Vote result thread:
+
+${community_vote_result_thread_url}
+
+The release candidate:
+
+https://dist.apache.org/repos/dist/dev/incubator/fury/${release_candicate_version}/
+
+This release has been signed with a PGP available here:
+
+https://downloads.apache.org/incubator/fury/KEYS
+
+Git branch for the release:
+
+https://github.com/apache/incubator-fury/tree/release-${release_candicate_version}
+
+Maven staging repo:
+https://repository.apache.org/content/repositories/orgapachefury-${maven_artifact_number}/
+
+Please download, verify, and test.
+
+The VOTE will pass after 3 binding approve.
+
+[ ] +1 approve
+[ ] +0 no opinion
+[ ] -1 disapprove with the reason
+
+To learn more about apache fury, please see https://fury.apache.org/
+
+Checklist for reference:
+
+[ ] Download links are valid.
+[ ] Checksums and signatures.
+[ ] LICENSE/NOTICE files exist
+[ ] No unexpected binary files
+[ ] All source files have ASF headers
+[ ] Can compile from source
+
+More detailed checklist please refer to:
+https://github.com/apache/incubator-fury/tree/main/ci/release.py
+
+To compile from source, please refer to:
+https://github.com/apache/incubator-fury/blob/main/CONTRIBUTING.md
+
+Thanks
+
+${name}
+```
+
+After at least 72 hours with at least 3 +1 binding vote (from Incubator PMC member) and no veto, claim the vote result:
+
+Title:
+
+```
+[RESULT][VOTE] Release Apache Fury(incubating) ${release_candicate_version}
+```
+
+Content:
+
+```
+Hi Incubator PMC,
+
+The vote to release Apache Fury(incubating) ${release_candicate_version} has passed with
+4 +1 binding and 3 +1 non-binding votes, no +0 or -1 votes.
+
+Binding votes：
+
+- xxx
+- yyy
+- zzz
+
+Non-Binding votes:
+
+- aaa
+
+Vote thread: ${incubator_vote_thread_url}
+
+Thanks for reviewing and voting for our release candidate.
+
+We will proceed with publishing the approved artifacts and sending out the announcement soon.
+```
+
+### What if vote fail
+
+If the vote failed, click "Drop" to drop the staging Maven artifacts.
+
```

Comment:
If there are some concerns raised from the vote thread, could we update current build artifacts  without filing a new vote thread if we don't update the code. 

For example, one may forget add signature by accident. Could we continute current vote if the release manager uploaded the signature file and reply in the vote thread